server/src/db/seeders/ReviewSeeder.js

```diff
@@ -4,30 +4,54 @@ class ReviewSeeder {
   static async seed() {
     const reviewData = [
       {
-        rating: "10",
+        rating: 10,
         description: "I have never seen a better dog. Seriously.",
         upvotes: 5,
         downvotes: 1,
         userId: 1,
         dogId: 2,
       },
       {
-        rating: "9",
+        rating: 9,
         description: "Wow.  What a dog!",
         upvotes: 3,
         downvotes: 2,
         userId: 3,
         dogId: 1,
       },
       {
-        rating: "14",
+        rating: 14,
         description:
           "I have been looking at pictures of dogs for over 30 years. This is the best-looking one yet.",
         upvotes: 5,
         downvotes: 1,
         userId: 2,
         dogId: 3,
       },
+      {
+        rating: 13,
+        description: "This is a fairly average dog. Not impressed.",
+        upvotes: 1,
+        downvotes: 4,
+        userId: 1,
+        dogId: 2,
+      },
+      {
+        rating: 11,
+        description: "Wow. Much bark. So floof.",
+        upvotes: 10,
+        downvotes: 22,
```

Comment:
wait, we might have missed something in your earlier PR. You shouldn't have upvotes and downvotes here. You don't have a voting feature yet, so these shouldnt even be on the Review column. 

Further, it's not necessarily a good idea to have upvotes and downvotes as columns, because they need to be computed. You would have to update a review every time a new vote is created or updated (assuming that a Vote record is a join between user and review). 

For now, can you remove this column from your database, and have a discussion as a group later about how you want to handle vote value storage when you are about to start the feature

Reply:
Ok -deleted.